proteus/SpatialTools.py

```diff
@@ -767,7 +767,7 @@ def __init__(self, domain, dim=(0., 0.), coords=(0., 0.), barycenter=None):
                         self.BC['y+'],
                         self.BC['x-']]
         # self.BC = BCContainer(self.BC_dict)
-        self.It = L**2+H**2/12
+        self.It = (L**2+H**2)/12
```

Comment:
The same for me, I never used that function because I was imposing inertia rather than calculating it.